enshrine-esdt-safe/tests/enshrine_esdt_safe_blackbox_test.rs

```diff
@@ -144,31 +202,11 @@ impl EnshrineTestState {
         }
     }
 
-    fn propose_set_unpaused(&mut self) {
-        self.world
-            .tx()
-            .from(ENSHRINE_ESDT_OWNER_ADDRESS)
-            .to(ENSHRINE_ESDT_ADDRESS)
-            .typed(enshrine_esdt_safe_proxy::EnshrineEsdtSafeProxy)
-            .unpause_endpoint()
-            .run();
-    }
-
-    fn propose_set_header_verifier_address(&mut self) {
-        self.world
-            .tx()
-            .from(ENSHRINE_ESDT_OWNER_ADDRESS)
-            .to(ENSHRINE_ESDT_ADDRESS)
-            .typed(enshrine_esdt_safe_proxy::EnshrineEsdtSafeProxy)
-            .set_header_verifier_address(HEADER_VERIFIER_ADDRESS)
-            .run();
-    }
-
-    fn propose_register_operation(&mut self, has_prefix: bool) {
+    fn propose_register_operation(&mut self, has_prefix: bool, tokens: &Vec<TestTokenIdentifier>) {
         let (tokens, data) = if has_prefix {
-            self.setup_payments(vec![PREFIX_NFT_TOKEN_ID, FUNGIBLE_TOKEN_ID])
+            self.setup_payments(tokens)
```

Comment:
same `self.setup_payments(tokens)` is both on if and else

Reply:
fixed